IPython/core/display.py

```diff
@@ -731,6 +730,9 @@ def __init__(self, data=None, url=None, filename=None, format=None,
                 if data[:2] == _JPEG:
                     format = self._FMT_JPEG
 
+        if not format:
+            format=self._FMT_JPEG
```

Comment:
We're actually already doing this a few lines below, but setting it to png instead. The problem came when the `.lower()` was added on the wrong side of the `None` check, causing the AttributeError.